src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -147,11 +147,15 @@ void SequentialSfMReconstructionEngine::RobustResectionOfImages(
       }
       while (badTrackRejector(4.0, nbOutliersThreshold) != 0);
       std::cout << "Bundle with " << bundleAdjustmentIteration << " iterations took " << std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now() - chrono_start).count() << " msec." << std::endl;
+      chrono_start = std::chrono::steady_clock::now();
```

Comment:
FYI, In order to be sure that Chrono is supported everywhere, you can use the openMVG system::Timer class.